styles/styles.css

```diff
@@ -0,0 +1,168 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  margin: 5px;
+}
+/* 
+#heading {
+  display: flex;
+  flex-wrap: nowrap;
+} */
+
+#startrly {
+  margin-right: 400px;
+  margin-left: 1px;
+}
+
+#try-now {
+  margin-left: 400px;
+  margin-right: 300px;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  font-weight: bold;
+}
+
+section:first-of-type {
+  text-align: center;
+}
+
+#tagline {
+  font-size: 1em;
+  display: block;
+  margin-bottom: 20px;
+  text-transform: capitalize;
+  font-weight: bold;
+}
+
+#actionbox { 
+  border: solid lightgrey thin;
+  border-radius: 8px;
+  text-decoration: none;
+  color: black;
+  line-height: 5em;
+  padding: 12px;
+  font-size: 1em;
+}
+
+.quote {
+  font-family: 'Times New Roman', Times, serif;
+  font-style: italic;
+  padding: 30px 15px 15px 15px;
+  text-align: center;
+  border-top: 0.25px solid lightgray;
+
+}
+
+blockquote:before {
+  content: open-quote;
+}
+
+blockquote:after {
+  content: close-quote;
+}
+
+.cite {
+  text-align: right;
+  padding-right: 200px;
+}
+
+
+#why {
+  padding-top: 30px;
+  display: flex;
+  flex-flow: row;
+  align-items: flex-start; 
+  list-style-type: none;
+}
+
+h2 {
+  text-align: center;
+  font-weight: bold;
+  font-size: 1.5rem;
+  border-top: 0.25px solid lightgray;
+  padding-top: 30px;
+}
+
+.image-item {
```

Comment:
This section of code is not needed, but I also could not figure out how to get rid of the gap while using flex for the ul that you id'd #why (in your case). Still trying to figure this one out myself! But I went with a grid for now.